TF_Tooling_Web/.gitignore

```diff
@@ -8,4 +8,4 @@
 ./test/**/*.js
 dist
 
-./Plugins/**/SnappingPlugin.js
+./Plugins/**/Snapping_Plugin.js
```

Comment:
Why has the snapping plugin been added to gitignore? Is this just removing a locally built version?